Rockerfile

```diff
@@ -0,0 +1,14 @@
+# build image
```

Comment:
the magic of Rocker is we can separate the build and runtime image.
the build image container all resource or dependencies required at build time.
once the build success, copy it to the runtime image which based on alpine.
so the final image is alpine + MailHog executable binary.